leaf-ui/js/object/Class.js

```diff
@@ -340,26 +355,36 @@ class AnalysisResult {
             this.timePointPath = analysisResult.timePointPath;
             this.timePointPathSize = analysisResult.timePointPathSize;
             this.elementList = analysisResult.elementList; 
-            this.allSolution = analysisResult.allSolution;
+            this.allSolution = analysisResult.allSolution; //potentially deprecated
             this.elementListPercentEvals = analysisResult.elementListPercentEvals;
             this.isPathSim = analysisResult.isPathSim ; //used for slider visualization
             this.colorVis = analysisResult.colorVis; //color visualization for analysis mode  
             this.selectedTimePoint = analysisResult.selectedTimePoint; //find where slider is initialized and set timepoint in here. Also place it in update function
-            this.maxTimePoint = analysisResult.maxTimePoint;
+            this.timeScale = analysisResult.timeScale;
         } else {
             // new default analysisResult
             this.assignedEpoch;
             this.timePointPath ;
             this.timePointPathSize;
             this.elementList; 
-            this.allSolution;
+            this.allSolution; //potentially deprecated
             this.elementListPercentEvals;
             this.isPathSim = false; //used for slider visualization
             this.colorVis; //color visualization for analysis mode  
             this.selectedTimePoint; //find where slider is initialized and set timepoint in here. Also place it in update function
-            this.maxTimePoint;
+            this.timeScale;
         }
     }
+
+    setTimeScale() {
+        this.timeScale = Number(this.timePointPath.length) - 1;
+
+    }
+
```

Comment:
Can you fix/reformat all the extra spacing here?

Reply:
Ok, I will do that! Thank you!

Reply:
> Can you fix/reformat all the extra spacing here?
Ok, I will do that! Thank you!